locales/en.json

```diff
@@ -322,5 +318,9 @@
   "Enter your province or territory": "Enter your province or territory",
   "On December 31, 2018, what was the province or territory of your home address?": "On December 31, 2018, what was the province or territory of your home address?",
   "Province or territory": "Province or territory",
-  "Province or territory of your home address": "Province or territory of your home address"
+  "Province or territory of your home address": "Province or territory of your home address",
+  "Check your mailing address": "Check your mailing address",
+  "Is this your correct address?": "Is this your correct address?",
+  "If this information doesn’t match your records, you can’t change it here.": "If this information doesn’t match your records, you can’t change it here.",
```

Comment:
we can delete line 324 & 325 since we're no longer using this

Reply:
good catch thanks!